### PR TITLE
Enable 'Embedding Dimensions' and 'Max Tokens' in all the tabs

### PR DESCRIPTION
**Pull request by sanjeev-kumar-4**
Display the 'Max Tokens' and 'Embedding Dimension' feature across languages and tasks, for all models. This addresses [Issue 1202](https://github.com/embeddings-benchmark/mteb/issues/1202) raised in the mteb repo.

### CODE REVIEW

File: refresh.py [hunk 1]
```diff
@@ -663,9 +663,9 @@ def refresh_leaderboard() -> tuple[list, dict]:
                 data_task_category = get_mteb_data(
                     tasks=[task_category], datasets=task_category_list
                 )
-                data_task_category.drop(
-                    columns=["Embedding Dimensions", "Max Tokens"], inplace=True
-                )
+                # data_task_category.drop(
+                #     columns=["Embedding Dimensions", "Max Tokens"], inplace=True
+                # )
```
Comment (KennethEnevoldsen) [inline refresh.py]:
```suggestion
```

File: refresh.py [hunk 2]
```diff
@@ -777,8 +777,8 @@ def load_results(data_path: str) -> list | dict | pd.DataFrame | str | None:
     print("Done calculating, saving...")
     # save them so that the leaderboard can use them.  They're quite complex though
     #   but we can't use pickle files because of git-lfs.
-    write_out_results(all_data_tasks, "all_data_tasks")
-    write_out_results(boards_data, "boards_data")
+    write_out_results(all_data_tasks, "all_data_tasks_temp2")
+    write_out_results(boards_data, "boards_data_temp2")
```
Comment (KennethEnevoldsen) [inline refresh.py]: Why the changed names?

**Review by KennethEnevoldsen — COMMENTED** [review]
Will you add a picture of the updated leaderboard - before and after as well

re. your question on auto-update. I believe the rest is auto-updated.

File: refresh.py (hunk 2 above)
Comment (sanjeev-kumar-4) [reply refresh.py]: Restored the original folder name

**Comment by sanjeev-kumar-4** [comment]
Here are before and after images for two task and language pair:

**Classification - English : Before**
classification-english-before

**Classification - English : After**
classification-english-after

**Pair Classification - Polish : Before**
pair_classification-polish-before

**Pair Classification - Polish : After**
pair_classification-polish-after

**Comment by Samoed** [comment]
@KennethEnevoldsen I think that this can be merged

**Comment by KennethEnevoldsen** [comment]
Yea!

**Pull request merged**